src/Employer/Employer.Web/Mappings/Extensions/IApprenticeshipProgrammeMapperExtensions.cs

```diff
@@ -3,9 +3,9 @@
 using Esfa.Recruit.Employer.Web.ViewModels.Part1.Training;
 using Esfa.Recruit.Vacancies.Client.Domain.Entities;
 
-namespace Esfa.Recruit.Employer.Web.Extensions
+namespace Esfa.Recruit.Employer.Web.Mappings.Extensions
 {
-    public static class IApprenticeshipProgrammeExensions
+    public static class IApprenticeshipProgrammeMapperExtensions
```

Comment:
Does ApprenticeshipProgrammes type need an interface?It has resulted in this class sounding interfacey 🦊